tt_metal/common/utils.cpp

```diff
@@ -16,8 +16,8 @@ namespace utils
 {
     bool run_command(const string &cmd, const string &log_file, const bool verbose)
     {
-        ZoneScoped;
```

Comment:
I don't understand why these are getting commented out

Reply:
These were added mostly at the time of slow dispatch when program cache was not fully up and running. Now they are not as relevant and clutter the tracy timeline without much benefit. 
Should I just remove them? I was thinking in case anyone wants bring them back they would be easier to find

Reply:
Most of the profiling markers being commented out are in the compilation path. It may be useful to keep these in

Reply:
The parent compile call is staying. I was thinking if we see anything fishy with the parent call, we can turn these back on. I am actually thinking of adding a "Verbosity" level to Tracy. That way we can set different profiling levels with command line options.

Reply:
verbosity is a good call!